src/multiplay.cpp

```diff
@@ -674,13 +692,33 @@ BASE_OBJECT *IdToPointer(UDWORD id, UDWORD player)
 	return nullptr;
 }
 
+bool isBlindPlayerInfoState()
+{
+	if (game.blindMode == BLIND_MODE::NONE)
+	{
+		return false;
+	}
+
+	// If blind lobby (only) and game hasn't started yet
+	if (game.blindMode < BLIND_MODE::BLIND_GAME && !ingame.TimeEveryoneIsInGame.has_value())
```

Comment:
Maybe explicitly check for needed enumeration values instead of relying on `<` or `>=`? This way it won't raise additional questions for a person reading the code (e.g. is there any reason to establish some ordering additional ordering on these values? Is it crucial for some other code path?)